src/Orleans/Orleans.csproj

```diff
@@ -40,10 +39,10 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjection)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
-    <PackageReference Include="System.ValueTuple" Version="$(SystemRuntimeVersion)" />
+    <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(BuildFlavor)' == 'Legacy' ">
+  <ItemGroup>
```

Comment:
super nit: these ItemGroups can nor be merged

Reply:
They're separated for better human readibility, and grouped by item type. This was not changed now, it was this way before. These cleanups should be done with the legacy removal IMHO. Changeset is minimized and focused on .Net Standard 2.0 support.

Reply:
They were separated because this group had a condition, which doesn't have anymore, so they now belong together now even before we kill the legacy build flavor.
Still, I don't care enough